client/my-sites/plans-features-main/hooks/use-plan-upgrade-credits.ts

```diff
@@ -0,0 +1,40 @@
+import { useSelector } from 'react-redux';
+import { getPlanDiscountedRawPrice } from 'calypso/state/sites/plans/selectors';
+import { getSitePlanRawPrice } from 'calypso/state/sites/plans/selectors/get-site-plan-raw-price';
+import isPlanAvailableForPurchase from 'calypso/state/sites/plans/selectors/is-plan-available-for-purchase';
+import type { PlanSlug } from '@automattic/calypso-products';
+
+/**
+ * Calculate available plan credits given a set of displayed plans
+ *
+ * @param {*} siteId the selected site id
+ * @param {string[]}  plans Plans that are considered for the given calculation
+ * @returns {number} The amount of credits available for the given plans
```

Comment:
Similarly - I think `The amount of credits available for the given plans` points me to a "summation". Anyway, minor.